### PR TITLE
[DOC] activation api doc, test=document_fix

### PR DESCRIPTION
**Pull request by qili93**
### PR types

Others

### PR changes

Docs

### Describe

Change the Docs of the following APIs:

paddle.nn.ELU
paddle.nn.functional.elu

paddle.nn.GELU
paddle.nn.functional.gelu

paddle.nn.ReLU
paddle.nn.functional.relu

paddle.nn.LogSigmoid
paddle.nn.functional.logsigmoid

paddle.nn.Softmax
paddle.nn.functional.softmax

paddle.nn.PReLU
paddle.nn.functional.prelu

paddle.nn.Hardtanh
paddle.nn.functional.hardtanh

### CODE REVIEW

**Comment by qili93** [comment]
paddle.nn.ELU

image

paddle.nn.functional.elu

image

**Comment by qili93** [comment]
paddle.nn.GELU

image

paddle.nn.functional.gelu

image

**Comment by qili93** [comment]
paddle.nn.Hardtanh

image

paddle.nn.functional.hardtanh

image

**Comment by qili93** [comment]
paddle.nn.LogSigmoid

image

paddle.nn.functional.logsigmoid

image

**Comment by qili93** [comment]
paddle.nn.ReLU

image

paddle.nn.functional.relu

image

**Comment by qili93** [comment]
paddle.nn.Softmax

image

paddle.nn.functional.softmax

image

**Comment by qili93** [comment]
paddle.nn.PReLU

paddle.nn.functional.prelu

image

**Review by TCChenlong — APPROVED** [review]
LGTM
